### PR TITLE
respect user permissions

### PR DESCRIPTION
**Pull request by ableuler**

### CODE REVIEW

File: src/project/Project.present.js [hunk 1]
```diff
@@ -388,11 +394,11 @@ class ProjectSettings extends Component {
   render() {
     return <Col key="settings" xs={12}>
       <Row>
-        <Col xs={12} md={10} lg={6}><ProjectTags {...this.props} /></Col>
+        <Col xs={12} md={10} lg={6}><ProjectTags {...this.props} readOnly={this.props.accessLevel < 40}/></Col>
```
Comment (ciyer) [inline src/project/Project.present.js]: Create constants for these magic numbers (accessLevel == [30, 40, etc.]).

File: src/project/Project.present.js [hunk 2]
```diff
@@ -388,11 +394,11 @@ class ProjectSettings extends Component {
...
         <Col xs={12} md={10} lg={6}><RepositoryUrls {...this.props} /></Col>
       </Row>
       <Row>
-        <Col xs={12} md={10} lg={6}><ProjectDescription {...this.props} /></Col>
+        <Col xs={12} md={10} lg={6}><ProjectDescription {...this.props} readOnly={this.props.accessLevel < 40}/></Col>
```
Comment (ciyer) [inline src/project/Project.present.js]: Create constants for these magic numbers (accessLevel == [30, 40, etc.]).

File: src/file/File.container.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ class JupyterNotebookContainer extends Component {
   }
 
   componentDidMount() {
-    this.getDetploymentUrl()
+    if (this.props.accessLevel >= 30) this.getDetploymentUrl()
```
Comment (ciyer) [inline src/file/File.container.js]: Create constants for these magic numbers (accessLevel == [30, 40, etc.]).

File: src/file/File.container.js [hunk 4]
```diff
@@ -57,22 +57,19 @@ class LaunchNotebookServerButton extends Component {
...
```
Comment (ciyer) [inline src/file/File.container.js]: Create constants for these magic numbers (accessLevel == [30, 40, etc.]).

File: src/project/Project.present.js (hunk 1 above)
Comment (ableuler) [reply src/project/Project.present.js]: Fixed in https://github.com/SwissDataScienceCenter/renga-ui/pull/198/commits/f20837df079db4edb9c51948581467dda187453c

File: src/project/Project.present.js (hunk 2 above)
Comment (ableuler) [reply src/project/Project.present.js]: Fixed in f20837d

File: src/file/File.container.js (hunk 3 above)
Comment (ableuler) [reply src/file/File.container.js]: Fixed in f20837d

File: src/file/File.container.js (hunk 4 above)
Comment (ableuler) [reply src/file/File.container.js]: Fixed in f20837d

**Pull request merged**